project-base/storefront/styles/globals.css

```diff
@@ -203,7 +213,7 @@ input[type='number'] {
 
 .Toastify__toast-theme--colored.Toastify__toast--error
     .Toastify__progress-bar-theme--colored.Toastify__progress-bar--error {
-    @apply !bg-redLight;
+    @apply !bg-[#f2a2a2];
```

Comment:
Same there with red color

Reply:
This is not part of the scope which should be solved here, because there is no design for Toasts yet. But I solved it temporarily by some adjustments. Let's wait with other changes for new design then we can adjust.